src/gui_tool/entry.py

```diff
@@ -55,12 +55,17 @@ def split_file(file_loc, mdi:MetaDataItem):
 
     while True:
         try:
-            context = SPContext(file_loc)
+            mdi.bb_fields = BBFields()
```

Comment:
do we need to delete the bb_fields from the beginning? Maybe doing so on each newly created split would allow the user to save the bboxes if there is only one split while still allowing this functionality

Reply:
Maybe not... But why is a video that already has bounding boxes, and doesn't need to be re-split at all, going through the splitter again? I thought the only time something would go through the splitter twice is because a human manually decided a video segment needs to be re-split?

Reply:
I think its possible (and very likely) that a video thats labelled gets tagged as not-labelled for whatever reason, and I wouldnt want to delete the bboxes accidentally. Its mostly a just to be safe thing tbh

Reply:
That kinda sounds like we could have a very serious bug that breaks fundamental assumptions of the pipeline...